grc/main.py

```diff
@@ -224,7 +224,7 @@ def get_config_file_path(config_file: str = "grc.conf") -> str:
 
 
 def get_state_directory() -> str:
-    oldpath = os.path.expanduser("~/.gnuradio")
+    oldpath = os.path.expanduser("~/.grc_gnuradio")
```

Comment:
Is this correct? Only compiled hierarchical blocks were stored in `~/.grc_gnuradio`; `grc.conf` and `grc_qt.conf` were in `~/.gnuradio`.

Reply:
Never mind; I see that there's a separate `get_config_file_path` function for the configuration files.